iati/tests/about_functional_tests.py

```diff
@@ -256,6 +257,8 @@ def test_can_create_case_study_page(self, admin_browser):
         publish_page(admin_browser)
         view_live_page(admin_browser, CASE_STUDY_PAGE['title'])
         assert admin_browser.is_text_present(CASE_STUDY_PAGE['title'])
+        admin_browser.visit(os.environ['LIVE_SERVER_URL'])
+        assert admin_browser.is_text_present("Case studies")
```

Comment:
Ideally this should be its own test as it's asserting something different to the rest of the test.